### ISSUE TITLE
failed to download database [GeoLite2-ASN.mmdb] error in ES logs 

### ISSUE THREAD

**Issue by bhavyarm** [elastic/elasticsearch#72275]
**Elasticsearch version** (`bin/elasticsearch --version`): 7.13.0 BC2

**OS version** (`uname -a` if on a Unix-like system): darwin_x86_64

**Description of the problem including expected versus actual behavior**: I downloaded BC2 7.13.0 es artifact and started bin/elasticsearch and I see these in the logs:

````
[2021-04-26T15:11:34,251][INFO ][o.e.i.g.GeoIpDownloader  ] [Bhavyas-MacBook-Pro.local] updating geoip databases
[2021-04-26T15:11:34,251][INFO ][o.e.i.g.GeoIpDownloader  ] [Bhavyas-MacBook-Pro.local] fetching geoip databases overview from [https://geoip.elastic.co/v1/database?elastic_geoip_service_tos=agree]
[2021-04-26T15:11:34,677][INFO ][o.e.i.g.GeoIpDownloader  ] [Bhavyas-MacBook-Pro.local] geoip database [GeoLite2-ASN.mmdb] is up to date, updated timestamp
[2021-04-26T15:11:36,340][INFO ][o.e.i.g.GeoIpDownloader  ] [Bhavyas-MacBook-Pro.local] geoip database [GeoLite2-City.mmdb] is up to date, updated timestamp
[2021-04-26T15:11:37,465][INFO ][o.e.i.g.DatabaseRegistry ] [Bhavyas-MacBook-Pro.local] downloading geoip database [GeoLite2-Country.mmdb] to [/var/folders/j8/hpsz_brs769_ks7xwfrynh280000gn/T/elasticsearch-2351452285725419569/geoip-databases/UvvLrmBZTTmaLIIpAC5y2A/GeoLite2-Country.mmdb.tmp.gz]
[2021-04-26T15:11:37,467][INFO ][o.e.i.g.DatabaseRegistry ] [Bhavyas-MacBook-Pro.local] downloading geoip database [GeoLite2-ASN.mmdb] to [/var/folders/j8/hpsz_brs769_ks7xwfrynh280000gn/T/elasticsearch-2351452285725419569/geoip-databases/UvvLrmBZTTmaLIIpAC5y2A/GeoLite2-ASN.mmdb.tmp.gz]
[2021-04-26T15:11:37,467][INFO ][o.e.i.g.DatabaseRegistry ] [Bhavyas-MacBook-Pro.local] downloading geoip database [GeoLite2-City.mmdb] to [/var/folders/j8/hpsz_brs769_ks7xwfrynh280000gn/T/elasticsearch-2351452285725419569/geoip-databases/UvvLrmBZTTmaLIIpAC5y2A/GeoLite2-City.mmdb.tmp.gz]
[2021-04-26T15:11:37,475][ERROR][o.e.i.g.DatabaseRegistry ] [Bhavyas-MacBook-Pro.local] failed to download database [GeoLite2-ASN.mmdb]
org.elasticsearch.ResourceNotFoundException: chunk document with id [GeoLite2-ASN.mmdb_4_1619464294683] not found
	at org.elasticsearch.ingest.geoip.DatabaseRegistry.lambda$retrieveDatabase$9(DatabaseRegistry.java:354) [ingest-geoip-7.13.0.jar:7.13.0]
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:673) [elasticsearch-7.13.0.jar:7.13.0]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) [?:?]
	at java.lang.Thread.run(Thread.java:831) [?:?]

```

**Steps to reproduce**:

Please include a *minimal* but *complete* recreation of the problem,
including (e.g.) index creation, mappings, settings, query etc.  The easier
you make for us to reproduce it, the more likely that somebody will take the
time to look at it.

 1. Download ES/kibana 7.13.0 BC2 from staging 
 2. Start ES and Kibana - bin/elasticsearch and bin/kibana
 3. ES logs show "failed to download database [GeoLite2-ASN.mmdb]"

**Comment by probakowski** [comment]
hey @bhavyarm, looks like something wrong happened with your cluster - it has no data it should have. Both this failure (missing chunk from `DatabaseRegistry`) and linked one from kibana shows that there's no data available or at least some indices (`.geoip_database`, `.kibana_task_manager_7.13.0_001` etc) are missing.
I'll add code in `GeoIpDownloader` that will redownload databases in such case but I wouldn't consider this a bug

**Comment by bhavyarm** [comment]
@probakowski apologies for the linked Kibana issue. I wanted them to know I had already logged a bug on download error in my kibana logs. Two separate issues. Thanks!

**Comment by thomasneirynck** [comment]
discussed this with @bhavyarm offline. Agreed that this seems unrelated to the Kibana issue, and that these are two separate issues.

This seems to point to an issue with ES retrieving the GeoIp data-files?

**Comment by probakowski** [comment]
This should be fixed now in 7.14 BC
